internal/blocksync/reactor.go

```diff
@@ -76,7 +77,7 @@ type Reactor struct {
 	stateStore sm.Store
 
 	blockExec   *sm.BlockExecutor
-	store       sm.BlockStore
+	store       store.BlockStore
```

Comment:
@sergio-mena , I reverted this temporarily to verify the behaviour of the rest of the code. Then we can re-implement your change and use an interface.

Reply:
I'm OK both ways. The change to using the interface was more of an exercise to make sure I had understood how the block storage is used.